base_geoengine/static/src/js/views/geoengine/layers_panel/layers_panel.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<templates>
+    <t t-name="base_geoengine.LayersPanel" owl="1">
+        <div
+            class="menu border-end"
+            t-attf-class="{{ state.isFolded ? 'o_layer_panel_fold ' : 'o_layer_panel bg-view' }}"
+        >
```

Comment:
not sure I like the formatting, but it seems quite readable indeed. is it the output of some tools like prettier?